multilep/test/sampleLists/17PrelimSamples.txt

```diff
@@ -0,0 +1,307 @@
+# DATA
+
+dilep:/DoubleEG/Run2017C-31Mar2018-v1/MINIAOD
+dilep:/DoubleEG/Run2017B-31Mar2018-v1/MINIAOD
+dilep:/DoubleEG/Run2017D-31Mar2018-v1/MINIAOD
+dilep:/DoubleEG/Run2017E-31Mar2018-v1/MINIAOD
+dilep:/DoubleEG/Run2017F-31Mar2018-v1/MINIAOD
+
+dilep:/DoubleMuon/Run2017B-31Mar2018-v1/MINIAOD
+dilep:/DoubleMuon/Run2017C-31Mar2018-v1/MINIAOD
+dilep:/DoubleMuon/Run2017D-31Mar2018-v1/MINIAOD
+dilep:/DoubleMuon/Run2017E-31Mar2018-v1/MINIAOD
+dilep:/DoubleMuon/Run2017F-31Mar2018-v1/MINIAOD
+
+dilep:/MET/Run2017B-31Mar2018-v1/MINIAOD
+dilep:/MET/Run2017C-31Mar2018-v1/MINIAOD
+dilep:/MET/Run2017D-31Mar2018-v1/MINIAOD
+dilep:/MET/Run2017E-31Mar2018-v1/MINIAOD
+dilep:/MET/Run2017F-31Mar2018-v1/MINIAOD
+
+dilep:/SingleElectron/Run2017B-31Mar2018-v1/MINIAOD
+dilep:/SingleElectron/Run2017C-31Mar2018-v1/MINIAOD
+dilep:/SingleElectron/Run2017D-31Mar2018-v1/MINIAOD
+dilep:/SingleElectron/Run2017E-31Mar2018-v1/MINIAOD
+dilep:/SingleElectron/Run2017F-31Mar2018-v1/MINIAOD
+
+dilep:/SingleMuon/Run2017B-31Mar2018-v1/MINIAOD
+dilep:/SingleMuon/Run2017C-31Mar2018-v1/MINIAOD
+dilep:/SingleMuon/Run2017D-31Mar2018-v1/MINIAOD
+dilep:/SingleMuon/Run2017E-31Mar2018-v1/MINIAOD
+dilep:/SingleMuon/Run2017F-31Mar2018-v1/MINIAOD
+
+dilep:/SinglePhoton/Run2017B-31Mar2018-v1/MINIAOD
+dilep:/SinglePhoton/Run2017C-31Mar2018-v1/MINIAOD
+dilep:/SinglePhoton/Run2017D-31Mar2018-v1/MINIAOD
+dilep:/SinglePhoton/Run2017E-31Mar2018-v1/MINIAOD
+dilep:/SinglePhoton/Run2017F-31Mar2018-v1/MINIAOD
+
+dilep:/Tau/Run2017B-31Mar2018-v1/MINIAOD
+dilep:/Tau/Run2017C-31Mar2018-v1/MINIAOD
+dilep:/Tau/Run2017D-31Mar2018-v1/MINIAOD
+dilep:/Tau/Run2017E-31Mar2018-v1/MINIAOD
+dilep:/Tau/Run2017F-31Mar2018-v1/MINIAOD
+
+#incomplete list of 2017 MC samples
+#to be further filled as samples become available
+
+#Drell-Yan
+dilep:/DYJetsToLL_M-50_TuneCP5_13TeV-amcatnloFXFX-pythia8/RunIIFall17MiniAOD-94X_mc2017_realistic_v10_ext1-v1/MINIAODSIM
```

Comment:
This is miniAODv1, we should only keep miniAODv2 for 2017 (i.e. delete lines like this and check also for the other samples in the list)